docs/contributor/architecture/README.md

```diff
@@ -43,11 +42,10 @@ called with respective resources they are managing. Once all is done, all resour
 are disposed and the same process is repeated for the next SKR. 
 
 The reconciler facing API like `Reconcile()` and `.SetupWithManager()` functions remains as close as possible to 
-the one defined by controller-runtime and used by Kubebuilder. 
+the one defined by controller-runtime and used by Kubebuilder.
 
 ![SKR Controller Manager](./assets/skr-controller-manager.drawio.svg)
```

Comment:
Do you have the steps in this diagram explained somewhere?

Reply:
not yet, I hope we will get there :)